src/components/shared/Loader.tsx

```diff
@@ -0,0 +1,10 @@
+export const Loader = (): JSX.Element => {
```

Comment:
https://github.com/DeFiCh/scan/blob/main/CONTRIBUTING.md#top-level-functions-should-not-use-arrow-syntax